velox/connectors/hive/HiveDataSink.h

```diff
@@ -227,6 +230,10 @@ class HiveInsertTableHandle : public ConnectorInsertTableHandle {
     return compressionKind_;
   }
 
+  std::unique_ptr<dwio::common::FlushPolicy> flushPolicy() {
+    return std::move(flushPolicy_);
```

Comment:
Subsequent invocation will return a nullptr.
Let's make `flushPolicy_` a `shared_ptr` and simplify.